specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/Databases.json

```diff
@@ -1104,16 +1040,30 @@
           "description": "Time in minutes after which database is automatically paused. A value of -1 means that automatic pause is disabled",
           "type": "integer"
         },
-        "storageAccountType": {
+        "currentBackupStorageRedundancy": {
```

Comment:
seems it is a rename? what is the reason?

Reply:
We are moving to this new name to be more clear for UX (what we discussed back in Dec)